modules/pipeline/services/pipelinesvc/batch_create.go

```diff
@@ -54,7 +55,8 @@ func (s *PipelineSvc) BatchCreate(batchReq *apistructs.PipelineBatchCreateReques
 		if batchReq.AutoRun {
```

Comment:
how about batch-create of cron but not auto run?

Reply:
cron is also compatibled in cron_task.go